pages/account/manage/event/[[...data]].js

```diff
@@ -336,11 +336,11 @@ export default function ManageEvent({ BASE_URL, event }) {
 
                 <div className="mt-6 flex items-center justify-end gap-x-6">
                   {event._id && (
-                    <Button type="button" onClick={() => setOpen(true)}>
+                    <Button type="button" onClick={() => setOpen(true)} style={{marginBottom: '1.5rem'}}>
```

Comment:
as someone already said, you can do this in tailwind unless you are calculating a value, you can just commit my change, should be faster for you, your choice
```suggestion
                    <Button type="button" onClick={() => setOpen(true)} className='mb-6'>
```